routing/index_graph_starter.hpp

```diff
@@ -48,14 +48,17 @@ class IndexGraphStarter final
     {
       return segment.GetMwmId() == m_segment.GetMwmId() &&
              segment.GetFeatureId() == m_segment.GetFeatureId() &&
-             segment.GetSegmentIdx() == m_segment.GetSegmentIdx();
+             segment.GetSegmentIdx() == m_segment.GetSegmentIdx() &&
+             (!m_strictForward || segment.IsForward() == m_segment.IsForward());
     }
 
     uint32_t GetSegmentIdxForTesting() const { return m_segment.GetSegmentIdx(); }
+    bool GetStrictForward() const { return m_strictForward; }
 
   private:
     Segment m_segment;
-    m2::PointD const m_point;
+    m2::PointD m_point;
```

Comment:
Either add default initialization for m_point to, or move default initialization for m_stringForward to the first ctor.